Marlin/Marlin_main.cpp

```diff
@@ -7413,6 +7554,20 @@ void process_next_command() {
           gcode_M0_M1();
           break;
       #endif // ULTIPANEL
+     
+      #if ENABLED(SPINDLE_ENABLE)
+        case 3:
```

Comment:
Note that other GCode commands include a brief synopsis on the `case N:` line. Feel free to add yours here.